src/index.js

```diff
@@ -31,15 +33,27 @@ function installFiles(sourceDir, done) {
   // The path to the package running the 'install' or 'postinstall' script.
   var fileInstallingPackagePath = hostPackageDir(scriptPath);
 
-  // The path to the package into which we should install the files.
-  var destinationDir = fileInstallingPackagePath && hostPackageDir(fileInstallingPackagePath);
-  if (!destinationDir) {
+  var source, target;
+  switch (npmv.majorVersion()) {
+    case '3':
```

Comment:
I was thinking about that too and came to the same conclusion -- I wasn't certain (and didn't look) to see if the version could be detected in v2.  Since I *knew* I could get a version from v3, that was the one I checked explicitly.

Similarly, no clue about v1 -- I think if you're still on that it might be time to upgrade before installing any other modules!